ui/src/error.ts

```diff
@@ -38,8 +39,14 @@ export const fromException = (exception: globalThis.Error): Error => {
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   const details = Object.assign({}, exception) as any;
   details.name = exception.name;
+
+  let code = Code.UnknownException;
+  if (details.name === "AbortError") {
```

Comment:
What is this `AbortError`? And where does it come from?

Reply:
The [`AbortError`](https://developer.mozilla.org/en-US/docs/Web/API/DOMException#exception-AbortError) is part of the AbortController API which can be used to abort a `fetch`. We use that to avoid request amplification and out-of-order returns leading to invalid state.